$NetBSD: patch-am,v 1.1 2008/03/18 11:48:25 dmcmahill Exp $

when building with f2c we need to define a dummy main.  This is typically
something like MAIN__.  This is not needed if f2c-f77 were used for linking
or if we'd used a different fortran compiler.

--- bem/src/nmmtl.cpp.orig	2004-07-28 13:50:22.000000000 +0000
+++ bem/src/nmmtl.cpp	2008-03-16 17:00:11.000000000 +0000
@@ -130,5 +130,10 @@
 
 
-
+#ifdef F77_DUMMY_MAIN
+extern "C" int F77_DUMMY_MAIN()
+{
+	return 0;
+}
+#endif
 
 int main (int argc, char **argv)
